### ISSUE TITLE
add / remove / Move sections via Gantry admin

### ISSUE THREAD

**Issue by shoulders** [gantry/gantry5#1649]
Hi,

platform = Joomla, V5.3.6

As an end user I would like the ability to add / remove / move sections on the layout page in the gantry admin without having to go and edit a file with an editor. This feature would be drag and drop just as with the other elements on the layout page.

This feature would allow people to be more creative and concentrate more on the design.

I know that you can add and move sections by editing a file (cant remember which).

I have looked through this repository and the wiki/agenda and could not find this suggested or mentioned just in-case i have missed it.

Thanks

shoulders

**Comment by shoulders** [comment]
reference #1499

**Comment by mahagr** [comment]
I see that you found the issue where we have already discussed about the subject. Lets continue the discussion in there to keep it all in a single location.

I agree that this is commonly asked question and something we should definitely look into. That said -- the first version of layout manager already did this, but it was seen as being too confusing approach (mostly by me and Kevin) and was replaced with a lot simpler solution.

You have probably already read the whole discussion and reasoning behind the simpler solution. All I know that this feature is not planned to be changed in the next version, but after that.. Maybe its time to look again how the layouts work and redesign that part.
